### ISSUE TITLE
Hi I got some swreveal problems

### ISSUE THREAD

**Issue by langsem** [John-Lluch/SWRevealViewController#750]
so when I launch my app, my simulator screen its just black, it won't perform the sw_front

**Comment by iDevelopper** [comment]
Is your segue class is SWRevealViewControllerSegueSetController for sw_front?

**Comment by langsem** [comment]
yh both of the controllers its set with a identifier sw_front and sw_rear

**Comment by iDevelopper** [comment]
Yes but with SWRevealViewControllerSegueSetController not SWRevealViewControllerSeguePushController?

**Comment by langsem** [comment]
![skjermbilde 2017-10-24 kl 18 51 55](https://user-images.githubusercontent.com/27305997/31956676-70425f6e-b8ec-11e7-8e3e-867ffff64094.png)

**Comment by iDevelopper** [comment]
I already seen your project. It worked???

**Comment by langsem** [comment]
yh but had to make better one with better construction

**Comment by langsem** [comment]
just forgot to set the sw as initial haha

**Comment by langsem** [comment]
What do I do when I get a error saying “error opening input file” bridging-header.h

Sendt fra min iPhone

24. okt. 2017 kl. 19:26 skrev Patrick <notifications@github.com<mailto:notifications@github.com>>:


I already seen your project. It worked???

**Comment by iDevelopper** [comment]
You have to go to build settings and specify the right place for this file:

![2017-10-31_08-09-45](https://user-images.githubusercontent.com/1782450/32211516-24359c72-be13-11e7-8a91-e4fa3ce561bb.png)

**Comment by langsem** [comment]
Have made the right path for it, copied the folder adress from the file and pasted it in, but it still won’t work

Sendt fra min iPhone

31. okt. 2017 kl. 08:12 skrev Patrick <notifications@github.com<mailto:notifications@github.com>>:


You have to go to build settings and specify the right place for this file:

[2017-10-31_08-09-45]<https://user-images.githubusercontent.com/1782450/32211516-24359c72-be13-11e7-8a91-e4fa3ce561bb.png>

**Comment by iDevelopper** [comment]
What is the path of this file?

**Comment by langsem** [comment]
thats the path to the bridging file


________________________________

**Comment by iDevelopper** [comment]
Yes OK, but what is it. You have to set a relative path from your project in the build settings.

**Comment by langsem** [comment]
It worked yes ! But one more question is it possible to have to bridging headers ? Cuz I have a jsonmessage files that needs bridging and now I have just imported both of them in the same bridging

Sendt fra min iPhone

31. okt. 2017 kl. 18:05 skrev Patrick <notifications@github.com<mailto:notifications@github.com>>:


Yes OK, but what is it. You have to set a relative path from your project in the build settings.

**Comment by iDevelopper** [comment]
You can import several files in the bridging header file.

**Comment by langsem** [comment]
Ok thanks :)

Sendt fra min iPhone

1. nov. 2017 kl. 07:15 skrev Patrick <notifications@github.com<mailto:notifications@github.com>>:


You can import several files in the bridging header file.
